monkey/monkey_island/cc/services/authentication_service/token/token_validator.py

```diff
@@ -16,3 +16,10 @@ def validate_token(self, token: Token):
         :raises SignatureExpired: If the token has expired
         """
         self._token_serializer.loads(token, max_age=self._token_expiration)
+
+    def get_token_payload(self, token: Token) -> str:
+        """
+        Returns the payload of a token
+        :param token: Token to get the payload of
+        """
+        return str(self._token_serializer.loads(token, max_age=self._token_expiration))
```

Comment:
Typically it's not the responsibility of the validator to decode something, but since we can't have `Manager` and `Service` (Mike doesn't allow us 😞) I don't know what's a better name for this.

Reply:
How about `TokenParser`. `TokenParser.parse()` can return a data structure containing the components of the token. It can raise validation errors if the token is invalid.